app/views/mentoring/automation/tooltip_locked.html.haml

```diff
@@ -1,8 +1,9 @@
-.heading.c-generic-tooltip.text-14{class:"!py-16 !px-24 w-[413px]"}
+.heading.c-generic-tooltip.text-14{class:"!py-16 !px-24 md:w-[413px] w-auto"}
```

Comment:
What is `.heading`?

Reply:
It was present when I arrived, assumed, it was a parent element to the others, but it is nothing, so just removed it.